site/content/blog/2023-09-25-postman-add-on/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Postman Add-on"
+summary: "Import Postman collections with the new Postman add-on for ZAP."
+type: post
+tags:
+- blog
+- postman
+- gsoc
+- gsoc-2023
+date: "2023-09-25"
+authors:
+    - vitikasoni
+---
+This extension allows us to easily import [Postman collections](https://www.postman.com/collection/) into ZAP, and hence we can then use ZAP’s powerful security testing capabilities to identify potential vulnerabilities in our APIs developed using Postman.
+
+## Installing the add-on
+
+The help page of the add-on can be found [here](https://www.zaproxy.org/docs/desktop/addons/postman-support/) and you can install it via the [ZAP Marketplace](https://www.zaproxy.org/addons/).
```

Comment:
The scheme and domain (`https://www.zaproxy.org`) should be removed, leave just the full path.